torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -83,7 +83,7 @@ int64_t SizeAdd::getDynamicValue() const {
   return dim_node_0->getDynamicValue() + dim_node_1->getDynamicValue();
 }
 
-std::string SizeAdd::ToString() const { return "SizeAdd"; }
+std::string SizeAdd::ToString() const { return "SizeAdd for op aten::add"; }
```

Comment:
are these debugging message that can be removed? It is a long tostring lol

Reply:
This string has to contain the op name `aten::add` otherwise it will fail to output the IR https://github.com/pytorch/xla/blob/ae35e1e2a808b1920123d76e71871a3a49f1cb58/torch_xla/csrc/ir_dump_util.cpp#L114.

Maybe I can just use `aten::add`. WDYT?

Reply:
let's just update the check 
```
 XLA_CHECK_NE(pos, std::string::npos) << node_string << " : " << op_string; 
```
to exclude the `SizeXXX` nodes. Make the check more flexible that it first check whether node is a SizeNode.

Reply:
Wouldn't it make the code hacky? It looks to me we don't use the "ToString()" much because it is not in the IR text, as you can see in the comment above. It seems the string in "ToString()" only shows up in the error message.

Reply:
ok then `aten::add_size` maybe?

Reply:
SG. Done